client/go/defaults/defaults.go

```diff
@@ -0,0 +1,251 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// Author: arnej
+
+package defaults
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/vespa-engine/vespa/client/go/trace"
+)
+
+const (
+	ENV_VESPA_HOME = "VESPA_HOME"
+	ENV_VESPA_USER = "VESPA_USER"
+	ENV_VESPA_HOST = "VESPA_HOSTNAME"
+
+	ENV_CONFIGSERVERS     = "VESPA_CONFIGSERVERS"
+	ENV_ADDR_CONFIGSERVER = "addr_configserver"
+
+	DEFAULT_VESPA_HOME = "/opt/vespa"
+	DEFAULT_VESPA_USER = "vespa"
+	DEFAULT_VESPA_HOST = "localhost"
+
+	DEFAULT_VESPA_PORT_BASE = 19000
+	ENV_VESPA_PORT_BASE     = "VESPA_PORT_BASE"
+
+	CONFIGSERVER_RPC_PORT_OFFSET = 70
+	ENV_CONFIGSERVER_RPC_PORT    = "port_configserver_rpc"
+
+	CONFIGPROXY_RPC_PORT_OFFSET = 90
+	ENV_CONFIGPROXY_RPC_PORT    = "port_configproxy_rpc"
+
+	DEFAULT_WEB_SERVICE_PORT = 8080
+	ENV_WEB_SERVICE_PORT     = "VESPA_WEB_SERVICE_PORT"
+)
+
+/**
```

Comment:
This looks like javadoc syntax. Should be simply 
```
/*
foo 
bar 
*/
```

https://go.dev/doc/comment